modules/ROOT/pages/securing.adoc

```diff
@@ -3,12 +3,66 @@ ifndef::env-site,env-github[]
+
+To provide secure access to information, applications and services can apply a variety of security measures. Mule runtime engine provides several tools and methods that enable developers to protect applications according to security requirements, prevent security breaches and facilitate authorized access to data.
+Going from securing your application configuration properties to using cryptography capabilities, configuring FIPS 140-2 certified environment, securing flows with Spring Security, configuring TLS cryptographic protocol, obtaining access to protected resource using Oath Authorization Grant Types and configuring the Mule Secure Token Service.
+
+
+== Secure Configuration Properties
+Encrypt configuration properties for your applications. The process involves creating a secure configuration properties file, defining the secure properties in the file and configuring the file in your project with the Mule Secure Configuration Properties Extension module. +
+
+See details in xref:secure-configuration-properties.adoc[Secure Configuration Properties]
+
+== Cryptography Module
+The module provides cryptography capabilities to a Mule application. Main capabilities include:
+
+* Symmetric encryption and decryption of messages.
+* Asymmetric encryption and decryption of messages.
+* Message signing and signature validation of signed messages. +
+
+This module supports three different strategies to encrypt and sign your messages:
+
+* xref:cryptography-pgp.adoc[PGP] +
+ For signature/encryption.
+* xref:cryptography-xml.adoc[XML] +
+ For signature/encryption of XML documents or elements.
+* xref:cryptography-jce.adoc[JCE] +
+ For using a wider range of cryptography capabilities provided by the Java Cryptography Extension. +
+
+See details in xref:cryptography.adoc[Cryptography Module]
+
+== FIPS 140-2 Compliance Support
+Mule 4 can be configured to run in a FIPS 140-2 certified environment. There are two requirements:
+
+* Have a certified cryptography module installed in your Java environment
+* Adjust Mule settings to run in FIPS security mode
+
+See details in xref:fips-140-2-compliance-support.adoc[FIPS 140-2 Compliance Support]
+
+== Spring Security
+Spring Security provides a number of authentication and authorization providers such as JAAS, LDAP, CAS (Yale Central Authentication service), and DAO. The following topics help get you started securing your flows using Spring Security:
+
+* xref:setting-up-ldap-provider-for-spring-security.adoc[Configure LDAP Provider for Spring Security] +
+Perform component authorization, or use it as a Mule security provider.
+* xref:component-authorization-using-spring-security.adoc[Component Authorization Using Spring Security] +
+Configure authorization using Spring Security features on your Mule components, so that users with different roles can only invoke certain methods.
```

Comment:
```suggestion
Configure authorization using Spring Security features on your Mule components, so that users with different roles can invoke only certain methods.
```